spec/support/spec_test_helper.rb

```diff
@@ -7,3 +7,9 @@ def login
     login_as(create(:user))
   end
 end
+
+def stub_fixture(url, filename)
+  url = url.gsub(/\#cmt\d+$/, '')
```

Comment:
Maybe we should parse the URL here using `URI` and then remove the whole fragment, to make it more general?